index.js

```diff
@@ -1,3 +1,23 @@
+class Room {
+  constructor() {
+  }
+};
```

Comment:
If you pass arguments to your constructor you can generate your objects in a valid state by calling `new Room(arg1, arg2)` and passing the values you want to correspond to the keys as arguments.